isolated-contexts.bs

```diff
@@ -296,17 +306,19 @@ subresources</dfn> if the following algorithm returns "`Sufficient`":
 
 <div algorithm="trusted type requirements">
 A [=policy=] |policy| <dfn for="policy">sufficiently mitigates DOM sinks</dfn>
-if the following algorithm returns "`Sufficient`":
+if the following algorithm returns "`Sufficient`".
+Possible return values are "`Sufficient`" and "`Not sufficient`".
 
 <ol class="algorithm">
     1.  [=For each=] |directive| in |policy|'s [=policy/directive set=]:
 
         1.  Return "`Sufficient`" if all of the following are true:
 
             *   |directive|'s [=directive/name=] is
-                "`require-trusted-types-for`". [[!TRUSTED-TYPES]]
+                "`<a href="https://w3c.github.io/trusted-types/dist/spec/#require-trusted-types-for-directive">require-trusted-types-for</a>`".
+                [[!TRUSTED-TYPES]]
```

Comment:
Add this to the `<pre class="anchors">` section and file an issue with the Trusted Types spec to have this definition exported:

```
urlPrefix: https://w3c.github.io/trusted-types/dist/spec/; spec:trusted-types
    type: dfn
        text: require-trusted-types-for-directive
```
```suggestion
                "[=require-trusted-types-for-directive|require-trusted-types-for=]".
```

Reply:
Thanks, filed https://github.com/w3c/trusted-types/issues/538